aten/src/ATen/native/native_functions.yaml

```diff
@@ -4014,6 +4014,9 @@
     MkldnnCPU: mkldnn_clone
     QuantizedCPU, QuantizedCUDA: quantized_clone
 
+- func: positive(Tensor self) -> Tensor
```

Comment:
`func: positive(Tensor(a) self) -> Tensor(a)` ?

Reply:
Fixed it and confirmed jit test has passed locally. Thanks!